src/main/java/Main.java

```diff
@@ -0,0 +1,66 @@
+import feign.Feign;
+import feign.jackson.JacksonDecoder;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.cloud.openfeign.support.SpringMvcContract;
+import uk.gov.hmcts.reform.fees.client.FeesApi;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+@SpringBootApplication
+@EnableFeignClients
+public class Main {
```

Comment:
Please remove this class and add mock tests, we are using WireMock for other similar Feign clients.

Reply:
Will not remove yet as this will be used for QA testing, but will add WireMock tests.

Reply:
It's ok for testing, but please remove before merging it.